hphp/ppc64-asm/asm-ppc64.h

```diff
@@ -784,11 +789,12 @@ struct Assembler {
   void call(T& target, CallArg ca = CallArg::Internal) {
     if (CallArg::Internal == ca) {
       // tries best performance possible
-      branchAuto(target, BranchConditions::Always, LinkReg::Save);
+      branchAuto(target, BranchConditions::Always, LinkReg::Save, true);
     } else {
       // branchFar is not only smashable but also it will use r12 so an
       // external branch can correctly set its TOC address appropriately.
-      branchFar(target, BranchConditions::Always, LinkReg::Save);
+      branchFar(target, BranchConditions::Always, LinkReg::Save,
+                ImmType::TocOnly, true);
```

Comment:
Why are calls always `immMayChange`?

Reply:
The ```immMayChange``` is used to optimize the TOC allocation, avoiding duplicated values on it, but it cannot be used to immediate that may be changed when patching or smashing.
In this case I was defensive, forcing all calls to allocate one TOC slot, avoiding be referenced by another part of the code that may change it.